tools/plugins/ng-lazy.ts

```diff
@@ -27,31 +27,36 @@ export class NgLazyPluginClass {
 
   public bundleStart(context: WorkFlowContext) {
```

Comment:
Can you make it so that nested modules have their filename in the form of:

parent.module
parent-submodule.module

etc.... 

the reason is, someone can have another module with the same name unrelated to the nested modules and it would get deleted